### DISCUSSION TITLE
Additional language modes

### DISCUSSION THREAD

**Discussion by bebopagogo** [eteran/nedit-ng#346] [Q&A]
Are there additional language mode definitions available for nedit located somewhere aside from the standard ones included?  I am interested in CMake and JSON would be a useful one, too.  I am not a CMake expert which is why I am interested in a syntax highlighter for that.  Otherwise, I would consider learning how to create it myself.

**Comment by eteran** [comment]
There isn't anything official. I am hoping to develop a method for better sharing of the language definitions. Currently they are all in a large YAML file, but would probably be better as individual files in a directory. The only real complication there is that language settings (tab preferences and such) and language syntax highlighting are related, but defined separately, so I'll just need to figure out the best way to sort that all out.
